main/http_server/axe-os/src/app/components/swarm/swarm.component.html

```diff
@@ -40,8 +50,8 @@
                 <td>{{axe.hashRate * 1000000000 | hashSuffix}}</td>
                 <td>{{axe.uptimeSeconds | dateAgo}}</td>
                 <td>{{axe.sharesAccepted | number: '1.0-0'}}</td>
-                <td>{{axe.power | number: '1.2-2'}} <small>W</small> </td>
-                <td>{{axe.temp}}°<small>C</small></td>
+                <td>{{axe.power | number: '1.1-1'}} <small>W</small> </td>
+                <td [ngClass]="{'text-orange-500': axe.temp > 65}">{{axe.temp | number: '1.0-1'}}°<small>C</small></td>
```

Comment:
Nit: 66 and above feels a bit too aggressive for me, maybe 68 instead?
![image](https://github.com/user-attachments/assets/f914c3ec-92ba-43aa-aa5a-b455ee9e3ec4)